WP_Auth0.php

```diff
@@ -123,6 +123,19 @@ public function init() {
 		WP_Auth0_Email_Verification::init();
 	}
 
+  /**
+   * Is the Auth0 plugin ready to proc
```

Comment:
if "proc" is the short for something I didn't get it. To process, maybe?

Reply:
Maybe I just fell asleep in the middle of that docblock or something ...